src/utils/helpers/log_setup.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+
+def get_log():
```

Comment:
i can't see debug level logs in the application. Can you add some debug level logs?

Reply:
added debug logs in the code blocks.